modules/client/src/main/scala/lucuma/itc/client/json/WavelengthJson.scala

```diff
@@ -0,0 +1,14 @@
+// Copyright (c) 2016-2022 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package lucuma.itc.client.json
+
+import io.circe.*
+import io.circe.syntax.*
+import lucuma.core.math.Wavelength
+
+implicit val EncoderWavelength: Encoder[Wavelength] =
```

Comment:
`given`?

Reply:
Right, I switched `BandNormalized` and meant to go back and do the rest.  I'll do that before merging.